src/pprof/ddprof_pprof.cc

```diff
@@ -201,17 +201,15 @@ DDRes pprof_aggregate(const UnwindOutput *uw_output,
   // number of labels at present
   ddprof_ffi_Label labels[PPROF_MAX_LABELS] = {};
   size_t labels_num = 0;
```

Comment:
I believe those suppressions are needed.

Reply:
I still have something that went wrong with cppcheck, trying to figure that one out.